src/Microsoft.Framework.Runtime/NuGet/Xml/PackageMetadataXmlExtensions.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace NuGet.Xml
+{
+    public static class PackageMetadataXmlExtensions
+    {
+        private const string References = "references";
+        private const string Reference = "reference";
+        private const string Group = "group";
+        private const string File = "file";
+        private const string TargetFramework = "targetFramework";
+        private const string FrameworkAssemblies = "frameworkAssemblies";
+        private const string FrameworkAssembly = "frameworkAssembly";
+        private const string AssemblyName = "assemblyName";
+        private const string Dependencies = "dependencies";
+
+        public static XElement ToXElement(this ManifestMetadata metadata, XNamespace ns)
+        {
+            var elem = new XElement(ns + "metadata");
+            elem.SetAttributeValue(ns + "minClientVersion", metadata.MinClientVersionString);
```

Comment:
again - I don't think you want the attribute in the namespace (review everywhere). You could also move attribute creation to the ctor of the owning element (as well as children element).